internal/sdkv2provider/schema_cloudflare_access_identity_provider.go

```diff
@@ -136,7 +137,6 @@ func resourceCloudflareAccessIdentityProviderSchema() map[string]*schema.Schema
 					},
 					"redirect_url": {
 						Type:     schema.TypeString,
-						Optional: true,
```

Comment:
Context: redirect_url is always set by the API. Values sent by the client are ignored.

Reply:
should this be `Computed` then?

Reply:
it is 😛

Reply:
thanks GitHub diffs 🤦‍♂️